meson.build

```diff
@@ -0,0 +1,34 @@
+project('lgi', 'c',
+  version: '0.9.2',
+  meson_version: '>= 0.44.0',
+  default_options: [
+    'warning_level=2',
```

Comment:
According to https://github.com/mesonbuild/meson/blob/master/docs/markdown/Builtin-options.md, this should be `warnlevel`...?
Also, I have no idea what this option actually does... but whatever.

Reply:
Looks like both are valid, I've never seen `warnlevel` actually used, discussing with upstream if that should be more clearly documented.

It translates to `-Wall -Wextra` with `gcc/clang`